executor/src/constant_evaluator/mod.rs

```diff
@@ -616,8 +626,10 @@ mod test {
     pub fn arrays_of_fixed() {
         let src = r#"
             namespace F(4);
-                let x: int -> col = |k| |i| i + k;
+                let x: fe -> (int -> fe) = |k| |i| std::convert::fe(i) + k;
                 let y: col[2] = [x(0), x(1)];
+            namespace std::convert(4);
+                let fe = || fe();
```

Comment:
`std::convert::fe` is a built-in symbol. We still have to declare it (because I don't think it's a good idea to also clutter name resolution with builtins), but the actual semantics is replaced during type-checking and execution.
We can maybe find a better solution at some point.